doc/sphinx/source/changelog.rst

```diff
@@ -3,6 +3,112 @@
 Changelog
 =========
 
+.. _changelog-v2-11-0:
+
+v2.11.0
+-------
+Highlights
+
+- New CMORizers have been added:
+
+  - Updating ESACCI-WATERVAPOUR cmorizer (:pull:`3282`) by :user:`malininae`
+  - CMORiser for ANU Climate 2.0 Australian data (:pull:`3511`) by :user:`flicj191`
+
+- NEW TREND: First time release manager shout-outs!
+
+  - This is the first ESMValTool release managed by the Met Office! We want to
+    shout this out - and for all future first time release managers to
+    shout-out - to celebrate the growing, thriving ESMValTool community.
+
+This release includes
+
+Bug fixes
+~~~~~~~~~
+
+-  Update OceanSODA-ETHZ CMORizer with new source file (:pull:`3535`) by :user:`TomasTorsvik`
+-  Fix for ``recipe_seaice_drift.yml``: fix CRS transformer for "North Pole Stereographic" (:pull:`3531`) by :user:`flicj191`
+-  Recipe_ocean_quadmap: Update ATSR to match ESGF name (:pull:`3443`) by :user:`rbeucher`
+-  Fixed attribute handling in austral_jet/main.ncl for iris>=3.8 (:pull:`3603`) by :user:`schlunma`
+-  Update the name of the remapcon2 operator in R recipes (:pull:`3611`) by :user:`ehogan`
```

Comment:
These are also upgrades so the older code works with new dependencies, maybe it makes sense to have this in one place? Now half of these entries are under `Bug fixes` and the others are under `Diagnostics`

Reply:
Should these all be moved to `Installation` instead? 🤔

Reply:
How about adding a new section `Updates` or `Dependency updates`?

Reply:
Done in 5ecc2108c; did I miss any PRs? 🤔

Reply:
Maybe this one: https://github.com/ESMValGroup/ESMValTool/pull/3695#discussion_r1662622138

Reply:
Done in ef8737b84 😊